src/nyc_trees/apps/survey/views.py

```diff
@@ -614,7 +614,7 @@ def _create_survey_and_trees(request, event=None):
         # Convert any floats into integers for all the integer fields in Tree
         for field in ('circumference', 'stump_diameter'):
             if field in tree_data:
-                tree_data[field] = int(float(tree_data[field]))
+                tree_data[field] = int(round(float(tree_data[field])))
```

Comment:
@hectcastro The data is coming in from the client as a string, not a float. (`"2.5"`, not `2.5`).
`float(...)` converts the string to a number.